dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -1677,6 +1677,188 @@ bool buildRowColumnFilter(gp_walk_info* gwip, RowColumn* rhs, RowColumn* lhs, It
     return true;
 }
 
+bool buildEqualityPredicate(execplan::ReturnedColumn* lhs,
+    execplan::ReturnedColumn* rhs,
+    gp_walk_info* gwip,
+    boost::shared_ptr<Operator>& sop,
+    Item_func::Functype funcType,
```

Comment:
I don't see that we change this argument so it should be const.

Reply:
Done.